agent/api/container.go

```diff
@@ -147,6 +148,12 @@ type Container struct {
 	// exposed outside of the package so that it's marshalled/unmarshalled in the
 	// the JSON body while saving the state
 	SteadyStateStatusUnsafe *ContainerStatus `json:"SteadyStateStatus,omitempty"`
+
+	createdAt  time.Time
```

Comment:
If we want to save these information in the state file, we may need to bump the version. Otherwise can you add tag `json:"-"` here.

Reply:
These fields won't be serialized because they're private.